src/coffee/card.coffee

```diff
@@ -5,43 +5,45 @@ payment = require './payment/src/payment.coffee'
 extend = require 'node.extend'
 
 class Card
-  cardTemplate: """
-  <div class="jp-card-container">
-      <div class="jp-card">
-          <div class="jp-card-front">
-                  <div class="jp-card-logo jp-card-visa">visa</div>
-                  <div class="jp-card-logo jp-card-mastercard">MasterCard</div>
-                  <div class="jp-card-logo jp-card-amex"></div>
-                  <div class="jp-card-logo jp-card-discover">discover</div>
-              <div class="jp-card-lower">
-                  <div class="jp-card-shiny"></div>
-                  <div class="jp-card-cvc cjp-ard-display">{{cvc}}</div>
-                  <div class="jp-card-number jp-card-display">{{number}}</div>
-                  <div class="jp-card-name jp-card-display">{{name}}</div>
-                  <div class="jp-card-expiry jp-card-display" data-before="{{monthYear}}" data-after="{{validDate}}">{{expiry}}</div>
-              </div>
-          </div>
-          <div class="jp-card-back">
-              <div class="jp-card-bar"></div>
-              <div class="jp-card-cvc jp-card-display">{{cvc}}</div>
-              <div class="jp-card-shiny"></div>
-          </div>
-      </div>
-  </div>
-  """
+  cardTemplate: '' +
```

Comment:
What's the motivation for switching to string concatenation here rather than using Coffeescripts built in multi-line strings?